src/main/java/com/netgrif/application/engine/importer/service/FieldFactory.java

```diff
@@ -550,6 +558,15 @@ private void parseUserValues(UserField field, Case useCase, String fieldId) {
         field.setValue((UserFieldValue) useCase.getFieldValue(fieldId));
     }
 
+    private void parseUserListValues(Field field, Case useCase, String fieldId) {
```

Comment:
The implementation of similar methods for other fields takes as an argument the concrete class and not the generic `Field` class. This should be updated to fit with the rest of the implemented methods

Reply:
Resolved